$NetBSD: patch-be,v 1.2 2009/07/07 15:47:59 sborrill Exp $

sunpro couldn't resolve "find" without the prefix. Maybe there is some
method of that name?

--- src/Workspace.cc.orig	2008-01-15 09:50:53.000000000 +0000
+++ src/Workspace.cc	2009-07-06 12:47:47.000000000 +0100
@@ -85,7 +85,7 @@
 
 void Workspace::addWindow(FluxboxWindow &w) {
     // we don't need to add a window that already exist in our list
-    if (find(m_windowlist.begin(), m_windowlist.end(), &w) != m_windowlist.end())
+    if (std::find(m_windowlist.begin(), m_windowlist.end(), &w) != m_windowlist.end())
         return;
 
     w.setWorkspace(m_id);
